### DISCUSSION TITLE
Question about luarocks.loader and loading time

### DISCUSSION THREAD

**Discussion by craigmac** [lumen-oss/rocks.nvim#616] [Q&A] [answered]
I have this line in my `nvim --startuptime test` file:

```
076.191  031.372  030.037: require('luarocks.loader')
```

While I find the startuptime format very difficult to understand, chatgpt tells me that this call is taking roughly 31ms, is this correct? Any way to reduce the time this call is taking?

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

For interpreting the output of `--startuptime`, see `:h profd`.
Your output is measuring roughly 30 ms, but keep in mind that profiling itself adds overhead, so the actual time spent is likely less.

You can disable the luarocks loader with the `enable_luarocks_loader` option, see `:h rocks.config`.

**Reply by craigmac** [reply]
Neat, thanks! Second question, if I may, I set `vim.loader.enable()` in my init.lua, is that at odds/overlap what the luarocks loader is doing?

**Reply by mrcjkb** [reply]
no :)

**Reply by ghost** [reply]
So I shouldn't use `vim.loader.enable()` when using rocks.toml? It seems my startuptime decreases when I have it in my init.lua

**Reply by mrcjkb** [reply]
I doubt that `vim.loader.enable()` will cause problems. But keep in mind that it is an experimental feature.
